sql/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,14 @@
+-- This file creates a new table, subscriptions, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscription (
+    name varchar(50) NOT NULL,
+    subscription varchar(255) NOT NULL,
```

Comment:
Sooo I'm actually not sure what a `subscription` is or how this table is going to be used, but `subscription.subscription` is kind of bad form - is there any way to make that more identifyable

Reply:
I was going to have `subscriptions.subscription`.  I could just change it to `room`

Reply:
changed to `room`